introduction-to-calmm.md

```diff
@@ -878,7 +878,8 @@ using [`karet`](https://github.com/calmm-js/karet) and assuming that the `value`
 property will be an Atom:
 
 ```jsx
-const TextInput = ({value}) => <input {...U.bind({value})}/>
+const TextInput = ({value}) =>
```

Comment:
Not much of a one-liner anymore, heh

Reply:
:)

One could use [`U.Input`](https://github.com/calmm-js/karet.util#U-Input) there:

```js
const TextInput = ({value}) => <U.Input type="text" value={value}/>
```

But it is probably better to avoid using too many utilities in an introduction even if they are really simple.